src/qttableview.cpp

```diff
@@ -1040,18 +967,18 @@ void QtTableView::paintEvent(QPaintEvent *e)
     if (!colXPos(firstCol, &xStart))
     {
         // right empty area of table
-        // paint.eraseRect( updateR ); // erase area outside cells but
+        // p.eraseRect( updateR ); // erase area outside cells but
```

Comment:
Why changing the commented code?

Reply:
Find-and-replace. Not important because there's a long long way to a clean code.